test/functionalTest/cases/1126_GET_v2_subscriptions/GET_v2_all_subscriptions.test

```diff
@@ -0,0 +1,56 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+GET Subscriptions V2
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. GET /v2/subscriptions
+#
+
+echo "01. GET /v2/subscriptions"
+echo "========================="
+orionCurl --url /v2/subscriptions?options=count --in json --out any
```

Comment:
I know (I think I added 'text' and 'any' options). but I can't see the way of checking that our JSON does not contain spaces, new lines, ... if the beautifier changes the output. Maybe all test should be changed this way or some day we can be adding white spaces in the JSON we generate and passing the test ...